Examples/UICatalog/Scenarios/EditorsAndHelpers/DimEditor.cs

```diff
@@ -93,7 +93,7 @@ private void DimEditor_Initialized (object? sender, EventArgs e)
         };
         Add (label);
         _dimRadioGroup = new () { X = 0, Y = Pos.Bottom (label), RadioLabels = _radioItems };
-        _dimRadioGroup.SelectedItemChanged += OnRadioGroupOnSelectedItemChanged;
+        _dimRadioGroup.SelectedItemChanged += OnRadioGroupOnSelectedtemChanged;
```

Comment:
Typo `OnRadioGroupOnSelectedtemChanged` should be `OnRadioGroupOnSelectedItemChanged`.

Reply:
I just fixed all these.